### ISSUE TITLE
Clicking menubar icon at very top of screen doesn't work

### ISSUE THREAD

**Issue by zacharykeeping** [p0deje/Maccy#236]
It appears that the Maccy dropdown doesn't seem to trigger when clicking the icon at the very edge of the screen. I usually click menubar icons after casually moving my mouse to the top edge of the screen but this usually means I'm failing to trigger Maccy unless I move it down and click it with more precision. No other menubar items or third-party icons appear to have this issue on my Mac.

This is on a MacBook Air M1 running Big Sur 11.2.

**Comment by p0deje** [comment]
Oh, this is a very interesting bug. I'll look into this. Thanks for reporting!

**Comment by zacharykeeping** [comment]
I'm wondering if screen size/resolution has something to do with it? I seem to only have the issue on my MBA with the built-in screen, but if I plug into my 24" monitor it works correctly. Odd.

**Comment by zacharykeeping** [comment]
To add to this... It looks like this is only happening to me when triggered using a trackpad. Using a Magic Mouse it works fine triggering it from the top of the screen, but using either the built in trackpad or a Magic Trackpad it fails to trigger. Really odd.

**Comment by p0deje** [comment]
I can confirm that it works fine when I use a mouse and not a trackpad. Weirdo.

**Comment by MrFabio** [comment]
This issue happens to me when clicking the icon on the laptop "Desktop/Screen". When clicking on the screen from my external monitor it works, but the one from the laptop doesn't.
If I disconnect the monitor, the icon now works as expected on the laptop.

TLDR;
Screen A - Laptop
Screen B - External Monitor

When B is connected (thunderbolt) the icon works on B but not on A.
When B is disconnected and only A is active, the icon works as expected.
Chaning the mouse/trackpad has no effect.

Using Maccy v0.20.1

**Comment by p0deje** [comment]
@zacharykeeping @MrFabio Can you please check if the issue still happens in recent Big Sur 11.4 (20F71). I no longer can reproduce it.

**Comment by MrFabio** [comment]
I was on Catalina when I experienced this issue. Now I'm with _Big Sur 11.4 (20F71)_ and can confirm that it works now!

**Comment by p0deje** [comment]
Ok, I'm going to close it for now then. Feel free to reopen if it still happens.
